src/interfaces/resource-interfaces/resource-interface.ts

```diff
@@ -1,6 +1,25 @@
-import { CertifyStatementIF } from '@/interfaces'
-
 // Interface to define the resource model example
+import { IncorporationAgreementTypeIF } from '@/interfaces'
```

Comment:
Do you think the steps should be imported here as well?

Reply:
You bet, as soon as we can confidently identify changes to the steps.